.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+<!--
+    Anything in <!-- and -> are comments and would not appear to us
+-->
+
+* [ ] I have read the [guidelines].
```

Comment:
Very patronising. We're not asking for a terms and conditions. I'd rather we didn't have this.